### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by prachi-sahani**
Created home page screen for the e-commerce website.

### Features included:
 - navigation 
 - hero section
 - categories of products
 - footer(quick links and contact info)

(out of the 14 files only 3 files are to be reviewed, rest are assets 🙂 )
 
### Preview - 
 - desktop
ecom - homepage

- tablets 
127 0 0 1_5500_(iPad)

- mobile
127 0 0 1_5500_(iPhone 12 Pro)

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  text-align: center;
+  cursor: pointer;
+  overflow-wrap: break-word;
+}
+
+.category-cards .card:hover {
+  transform: scale(1.05);
+}
+
+.category-cards .card .card-title {
```
Comment (abhij1607) [inline homepage.css:19]: I've got a question, whether the extra specificity is required here?

**Review by abhij1607 — APPROVED** [review]
Nice work! really liked the class nomenclature

**Review by FarhanMobashir — COMMENTED** [review]
Everything is fine and good to go

File: index.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        <div class="footer-upper">
+            <ul class="footer-quick-links list-group txt-left">
+                <h4 class="headings h4">Quick Links</h4>
+                <li class="list-item"><a href="">Home</a></li>
+                <li class="list-item"><a href="">Cancellation & Refund Policy</a></li>
+                <li class="list-item"><a href="">Terms & Conditions</a></li>
+            </ul>
+            <ul class="footer-contact-links list-group txt-left">
+                <h4 class="headings h4">Get in touch</h4>
+                <li class="list-item"><i class="material-icons txt-bold txt-md pr-2">phone</i><a href="">(+91)
```
Comment (bhanuprakashj) [inline index.html:136]: add phone number prefixed with tel: in href by which on clicking it will open caller application

File: index.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                        9999999999</a></li>
+                <li class="list-item"><i class="material-icons pr-2">mail_outline</i><a
+                        href="">support@cakery.com</a></li>
```
Comment (bhanuprakashj) [inline index.html:139]: add mail id prefixed with mailto: in href by which it will open email client application

File: index.html (hunk 2 above)
Comment (Hamdrive) [reply index.html:136]: Wow this was something new I've learnt! Thanks Bhanu!

File: homepage.css (hunk 1 above)
Comment (prachi-sahani) [reply homepage.css:19]: Actually .card and .card-title are classes from component library(for all cards) which I didn't want to modify directly and hence this selector. I am not sure if this is the the right approach. Let me know your opinion.

**Pull request merged**